lib/cli/etcher.js

```diff
@@ -63,13 +63,25 @@ form.run([
   }, function(state) {
 
     if (options.robot) {
-      console.log([
-        'progress',
-        state.type,
-        Math.floor(state.percentage) + '%',
-        state.eta + 's',
-        Math.floor(state.speed)
-      ].join(' '));
+      if (process.send) {
```

Comment:
Document that `--robot` attempts to perform IPC communication is possible, before outputting to stdout.